docs/running-workflows/launching-workflows/index.md

```diff
@@ -135,22 +135,13 @@ The standard set of badges look as follows:
 
 ![Launch with Snakemake REANA at CERN](https://www.reana.io/static/img/badges/launch-with-snakemake-on-reana-at-cern.svg)
 
-If you would like to generate your own custom badge, you can use external services such as [shields.io](https://shields.io) or [badgen.net](https://badgen.net).
+To streamline the badge creation process, you can use the tool accessible on REANA at the
+[`/launcher-badge`](https://reana.cern.ch/launcher-badge) URL.
 
-### Markdown
-
-You can include one of the above badges into your Markdown documentation pages using the following syntax:
-
-```markdown
-[![Launch on REANA badge](https://www.reana.io/static/img/badges/launch-on-reana.svg)](https://reana.cern.ch/launch?url=https%3A%2F%2Fgithub.com%2Freanahub%2Freana-demo-root6-roofit)
-```
-
-This example points to the RooFit analysis demo. You should replace the `url` to point to your own analysis sources.
-
-### HTML
-
-You can include one of the above badges into your HTML documentation pages using the following syntax:
+Upon visiting the launcher badge page, you will be able to generate the necessary markdown syntax to include the badge
+in your documentation. This tool also provides a generic URL for initiating the workflow, which can be integrated into
```

Comment:
Something like:

```suggestion
in your documentation. This tool also provides you the URL for executing your workflow, which can be integrated into
```

To emphasize that the tool generates the URL specific to a workflow, and not a generic one that needs to be customised. What do you think?